lte/gateway/python/mypy.ini

```diff
@@ -0,0 +1,19 @@
+[mypy]
+python_version = 3.8
+ignore_missing_imports = True
+show_error_codes = True
+warn_unused_configs = True
+ignore_errors = False
+exclude = (?x)(
+    ^python/magma/pipelined/ | 
+    ^python/magma/enodebd/ | 
+    ^python/magma/subscriberdb/protocols/diameter/ | 
+    ^python/magma/kernsnoopd/ | 
+    ^python/precommit.py$ | 
+    ^python/scripts/agw_health_cli.py$ |
+    ^python/load_tests.common.py$ |
+    ^python/scripts/generate_oai_config.py$ |
+    ^python/setup.py$ |
+    ^python/scripts/state_cli.py$ |
+    ^python/integ_tests/ 
+    )
```

Comment:
Why placing this file within `lte/gateway/python/` and not in the repository root?
If at all, the Github action (#12008) would probably expect it to be at the repository root in order to load it (not tested so far).

Reply:
_minor styling_: There are some trailing whitespaces in this section.

Reply:
I moved the file to the repository root :+1: